CesiumGltf/include/CesiumGltf/PropertyTablePropertyView.h

```diff
@@ -258,6 +259,34 @@ class PropertyTablePropertyView<ElementType, false>
         _stringOffsetType{PropertyComponentType::None},
         _stringOffsetTypeSize{0} {}
 
+  /**
+   * @brief Construct an instance pointing to data specified by a {@link PropertyTableProperty}, with an enum definition attached.
+   * Used for non-array or fixed-length array data.
+   *
+   * @param property The {@link PropertyTableProperty}
+   * @param classProperty The {@link ClassProperty} this property conforms to.
+   * @param size The number of elements in the property table specified by {@link PropertyTable::count}
+   * @param values The raw buffer specified by {@link PropertyTableProperty::values}
+   * @param pEnumDefinition A pointer to the enum definition used for this
+   * value.
+   */
+  PropertyTablePropertyView(
+      const PropertyTableProperty& property,
+      const ClassProperty& classProperty,
+      int64_t size,
+      std::span<const std::byte> values,
+      const CesiumGltf::Enum* pEnumDefinition) noexcept
```

Comment:
Sorry, this is my OCD talking -- it feels inconsistent that the `pEnumDefinition` is at the bottom for `PropertyTablePropertyView` , but comes right after `ClassProperty` in `PropertyTexturePropertyView` . Can we pick one? :')